### DISCUSSION TITLE
TransitionCountModel properties

### DISCUSSION THREAD

**Discussion by hnadeem2** [deeptime-ml/deeptime#286] [Q&A] [answered]
Hello!

I had some confusion in understanding the following properties of the Transition Count Model object:
```
property selected_state_fraction: The fraction of states represented in this count model.
property state_symbols_with_blank: Symbols for states that are represented in this count model plus a state -1 for states which are not represented in this count model.
```
I assumed that represented here meant visited. But it seems it is not the case? Can you clarify what it means?
For my case the dtraj  data only has 23 unique states, whereas the original clustering object used 100 states. I was expecting `selected_state_fraction` to give 23/100, but it still gives me 1.0.
I found however that the `visited_set` property gives 23.

Best,
Hassan

**Comment by clonker** [comment, marked as answer]
Hi Hassan,
this means that from your 100 states, only 23 have counts (check `state_histogram_full`). The fraction reflects how many of these states you end up using; typically you would only use states which are connected via a transition. Otherwise quantities like stationary distribution are not meaningful. Once you have made that subselection (see, e.g., `submodel_largest`), the `selected_state_fraction` should change. :)

Hope this helps and best,
Moritz
